WordPress/WordPressTest/ViewRelated/Post/Controllers/PostListConflictResolverTests.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+import XCTest
+
+@testable import WordPress
+
+class PostListConflictResolverTests: XCTestCase {
+
+    private var context: NSManagedObjectContext!
+    var postListViewController: PostListViewController!
+    var blog: Blog!
+
+    override func setUp() {
+        super.setUp()
+        context = TestContextManager().mainContext
+        blog = BlogBuilder(context).build()
+        postListViewController = PostListViewController.controllerWithBlog(blog)
+    }
+
+    override func tearDown() {
+        context = nil
+        TestContextManager.overrideSharedInstance(nil)
+        super.tearDown()
+    }
+
+    func testAlertPresentsCorrectOptions() {
+        let post = createConflictedPost()
```

Comment:
While testing the alert is good and valid I think that the most important thing to be testes is the logic, like:

1. Given a post in version conflict and I discard local, this entity discards the local copy
2. Given a post in version conflict and I discard the web version, this entity calls the `PostCoordinator` to update the post with the local revision

The latter might be tricky, and you would need to inject the PostCoordinator to be able to mock it.

Reply:
I believe the tests are correct now, please let me know if you have feedback.